src/formatDate.js

```diff
@@ -50,7 +50,50 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const OldDate = date.split(fromFormat[3]);
+  const newDate = [];
+  const oldFormat = {};
+  let year;
+
+  for (let i = 0; i < 3; i++) {
```

Comment:
```suggestion
  for (let i = 0; i < 3; i++) {
```
move 3 to the constant because it looks like a magic number